playbooks/deploy_ipmitool.yml

```diff
@@ -0,0 +1,7 @@
+- name: Deploy IPMI tools
+  hosts: bastion
+  vars:
+    SETUP_IPMI_TOOLS: "{{ setup_pxe_service | default(True) }}"
```

Comment:
This will cause SETUP_IPMI_TOOLS to be true if it is not defined. Is this expected?

Reply:
In the inventory is set to False

Reply:
@halleelyasaf What if it doesn't exist in the inventory? you still need to set it to false in the playbook itself

Reply:
OK